src/devices/DCMotor/DCMotor2PinNoEnable.cs

```diff
@@ -73,7 +73,7 @@ public override double Speed
                         Controller.Write(_pin1, PinValue.High);
                     }
 
-                    _pwm.DutyCycle = 1.0 + val;
+                    _pwm.DutyCycle = Math.Abs(val);
```

Comment:
are you seeing any specific issue? this value is negative here and pin1 is opposite to what you have for positive values which means duty cycle has to be 1-Math.Abs(val)` but since we know value is negative we can just `1+val`